planet.php

```diff
@@ -0,0 +1,62 @@
+<?php
+require 'vendor/autoload.php';
+
+include 'src/Planet.php';
+use PlanetApp\PlanetHydrator;
+
+$dsn = 'mysql:host=db; dbname=planet_collection';
+$username = 'root';
+$password = 'password';
+
+if(!isset($_GET['planetId'])){header('Location: ./index.php');}
+
+$db = new PDO($dsn, $username, $password);
+$planetId = $_GET['planetId'];
+$planet = PlanetHydrator::getPlanet($db, $planetId);
+
+
+
+?>
+
+<!DOCTYPE html>
+
+<html lang="en-GB">
+<head>
+    <title>Planet Plinky Plonk</title>
+</head>
+<body>
+<nav>
+    <a href="./index.php#planet-<?=$planetId?>"><button type="button"><</button></a>
+</nav>
+<header>
+<h1><?=$planet->getName()?></h1>
+</header>
+<main>
+    <img src="<?=$planet->getImageSrc()?>" alt="<?=$planet->getName()?>">
+    <?php if($planet->getFullName() != null) {echo '<h2>Full Name: '.$planet->getFullName().'</h2>';}?>
+    <p>
+        <?=$planet->getDescription()?>
+    </p>
+    <div class="details-table">
+        <p class="detail-table-heading">Planet type</p>
```

Comment:
for styling, I think it may be worth putting each set of data into their own divs?

Reply:
good idea but not needed for grid :)